src/components/calc.js

```diff
@@ -0,0 +1,51 @@
+// import React from 'react';
+// import './calc.css';
+
+// const Calculator = () => (
+//   <div className="calculator">
+//     <div className="calculator-display">
+//       <input type="text" value="0" readOnly />
+//     </div>
+//     <div className="calculator-buttons">
+//       {/* Your calculator buttons go here */}
+//     </div>
+//   </div>
+// );
+
+// export default Calculator;
+
+import React from 'react';
+import './calc.css';
+
+const Calculator = () => (
+  <div className="calculator">
+    <div className="calculator-display">
+      <input type="text" value="0" readOnly />
+    </div>
```

Comment:
- [ ] According to the project requirements, you're required to have at least one child component being used inside the Calculator.js file. It's for the purpose of practicing the use of props and more components usage. Kindly create at least one child component that is used inside this component and take a prop used inside of it. 👍🏽

<img width="885" alt="Screenshot 2023-09-28 at 5 39 16 PM" src="https://github.com/MarkYegon7/Math-Magician/assets/50941074/f2d9a2de-f660-4d6f-b286-b5aab8474a54">

## Suggestions:

- [ ] You could create calculator result component and render it in the place of the result section of the design and pass a result value to it as a prop 👍🏽
- [ ] Or, you create Button component that you could use in place of the button element 👍🏽

Reply:
Done